modules/nw-ovn-k-anp-nbctl.adoc

```diff
@@ -0,0 +1,496 @@
+//module included in the following assemblies:
+//
+//networking/network_security/AdminNetworkPolicy/nw-ovn-k-anp-troubleshooting.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="metrics-commands-anp_{context}"]
+== Using nbctl commands for ANP and BANP
+
+To troubleshoot an unsuccessful setup, start by looking at OVN Northbound database (nbdb) objects which include the `ACL`, `AdressSet`, and `Port_Group`. To view thee nbdb, you need to be inside the pod on that node to see what objects are in the node.
```

Comment:
```suggestion
To troubleshoot an unsuccessful setup, start by looking at OVN Northbound database (nbdb) objects including `ACL`, `AdressSet`, and `Port_Group`. To view the nbdb, you need to be inside the pod on that node to see what objects are in the node.
```
The second sentence reads funky to me because you tell users to "view" the ndbd, but then to "see" what objects are in the node. Seems that only one "view" or "see" is needed, but I am not sure how to rephrase. I'll leave that to you. It is also fine as is, I was just thinking.

Reply:
missing the word database.